src/PerfViewCollect/PerfViewCollect.csproj

```diff
@@ -40,33 +40,26 @@
     <Compile Include="..\PerfView\ClrStats.cs" />
     <Compile Include="..\PerfView\GcStats.cs" />
     <Compile Include="..\PerfView\JitStats.cs" />
+    <Compile Include="..\PerfView\RuntimeLoaderStats.cs" />
```

Comment:
Are you wanting to enable this via the PerfViewCollect tool?

Reply:
Eventually... yes. But for now, I'm settling for it just building cleanly. This change was required for that. The rest of these changes, are VS doing something I didn't want.